src/main/java/bio/terra/cli/command/cromwell/GenerateConfig.java

```diff
@@ -28,6 +30,25 @@ public class GenerateConfig extends BaseCommand {
       description = "Directory to put generated cromwell.conf in. Defaults to current directory.")
   public String dir;
 
+  // We create an exclusive ArgGroup because we require one of either
+  // workspace_bucket_name or google_bucket_name.
+  @CommandLine.ArgGroup(exclusive = true, multiplicity = "1")
+  WorkspaceOrGoogleBucketName workspace_or_google_bucket_name;
+
+  static class WorkspaceOrGoogleBucketName {
+    @CommandLine.Option(
+        names = "--workspace-bucket-name",
+        required = true,
+        description = "Terra workspace bucket to hold Cromwell log files.")
+    public String workspace_bucket_name;
+
+    @CommandLine.Option(
+        names = "--google-bucket-name",
```

Comment:
how about gs-url?

maybe it's just me but i think it's a little confusing which name (resource name vs cloud name) to use.

Reply:
It is challenging to make it super clear differentiating between a workspace reference and a google cloud storage url. I had a conversation with @mbookman and this is what we came up with.

I did update the description to make it hopefully more clear.

Reply:
We decided on "path" when we added GCS url support to create references:
https://github.com/DataBiosphere/terra-cli/blob/main/src/main/java/bio/terra/cli/command/resource/addref/GcsObject.java#L40

"bucket name" implies no gs:// prefix.   If we require the prefix we should call it something else.

Reply:
Thanks @cbookg. My changes here don't require the gs:// prefix. If the user enters a `--google-bucket-name` without that prefix, it will be automatically added.

I've updated the README to make that more clear (also removed the old line that says you need to manually edit the generated conf).